controllers/topics.controllers.js

```diff
@@ -1,7 +1,13 @@
-const {fetchTopics} = require('../models/topics.models')
+const {fetchTopics, fetchEndpoints} = require('../models/topics.models')
 
-module.exports.getTopics = (request, response) => {
+exports.getTopics = (request, response) => {
     fetchTopics().then((topics) => {
         response.status(200).send({topics})
     })
 }
+
+exports.getEndpoints = (request, response) => {
+    fetchEndpoints().then((data) => {
+        response.status(200).send(data)
+    })
+}
```

Comment:
This is mostly fine but have a think about the variable name `data` - is this descriptive enough and represent the data that it holds? As a reader of your code, what should I infer from the variable name `data`, or is there a name which could be clearer?